Neo4j.Driver/Neo4j.Driver/Preview/Mapping/BuiltMapper.cs

```diff
@@ -56,18 +56,18 @@ public void AddWholeObjectMapping(Func<IRecord, TObject> mappingFunction)
 
     public void AddMappingBySetter(
         MethodInfo propertySetter,
-        MappingSource mappingSource,
+        EntityMappingInfo entityMappingInfo,
         Func<object, object> converter = null)
     {
         // create the .As<TProperty> method we're going to use
         var propertyType = propertySetter.GetParameters()[0].ParameterType;
         var asMethod = _asGenericMethod.MakeGenericMethod(propertyType);
-        var getter = _mappingSourceDelegateBuilder.GetMappingDelegate(mappingSource);
+        var getter = _mappingSourceDelegateBuilder.GetMappingDelegate(entityMappingInfo);
         AddMapping(propertySetter, GetValue);
 
         object GetValue(IRecord record)
         {
-            var found = getter(record, out var value);
+            getter(record, out var value);
```

Comment:
Why aren't we checking if it is found anymore?

Reply:
```suggestion
            var _ = getter(record, out var value);
```
We could make the unused parameter more explicit

Reply:
I put in a better bit so that `found` is handled separately to getting a null value out (since if `found` is false we shouldn't assume anything about `value`).